packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx

```diff
@@ -6,20 +6,24 @@ import ShareButtonStyle, {
 
 type Props = {
   iconSize?: number
+  title?: string
 } & ShareButtonStyleProps &
   React.HTMLAttributes<HTMLButtonElement>
 
 const ShareButton: React.FC<Props> = ({
   children,
   onClick,
   iconSize,
+  title,
   ...otherProps
 }) => (
   <ShareButtonStyle
     data-testid="share-button"
     size={30}
     onClick={onClick}
     variant="blank"
+    title={title}
```

Comment:
`title` here will be spread in otherProps already so not needed here I guess

Reply:
I'm also using `title` for `aria-label`

Reply:
yes, but aria-label is okay like this, the title prop already exist in HTMLAttributes